src/physics/cam/micro_pumas_cam.F90

```diff
@@ -1380,6 +1380,7 @@ subroutine micro_pumas_cam_tend(state, ptend, dtime, pbuf)
    use tropopause,      only: tropopause_find, TROP_ALG_CPP, TROP_ALG_NONE, NOTFOUND
    use wv_saturation,   only: qsat
    use infnan,          only: nan, assignment(=)
+   use perf_mod
```

Comment:
I think to stay with best practices (or at least CAM's) it would be good to add use `, only: t_startf, t_stopf` here.

Reply:
Thanks for the feedback.  I just updated the code as requested.  I also found a couple of other instances of the use of the perf_mod module without the only clause.  I modified these as well.